modules/ocf/templates/munin/ocf-plugin-conf.erb

```diff
@@ -0,0 +1,3 @@
+[memory]
+env.committed_warning :<%= (0.5 * 1e6 * @memorysize_mb.to_f).to_i %>
+env.committed_critical :<%= (0.8 * 1e6 * @memorysize_mb.to_f).to_i %>
```

Comment:
The choice of limits here is loosely based on graphs from the previous times ocfweb had memory leaks on coma:

http://munin.ocf.berkeley.edu/static/dynazoom.html?plugin_name=ocf.berkeley.edu%2Fcoma.ocf.berkeley.edu%2Fmemory&start_iso8601=2015-12-29T12%3A10%3A54-0800&stop_iso8601=2016-01-10T12%3A10%3A54-0800&start_epoch=1451419854&stop_epoch=1452456654&lower_limit=&upper_limit=&size_x=800&size_y=400&cgiurl_graph=%2Fmunin-cgi%2Fmunin-cgi-graph

These should probably get adjusted to fit our needs more closely.